db/migrate/helpers/delete_content.rb

```diff
@@ -5,13 +5,18 @@ def self.destroy_documents_with_links(content_ids)
 
       Document.where(content_id: content_ids).find_each do |document|
         destroy_edition_supporting_objects(document.editions)
+        destroy_content_id_aliases_for_document(document)
```

Comment:
should we also include a description of this behaviour in the unit test for "destroy documents with links"?

Reply:
oh, sorry, this is a migration helper, and doesn't have a unit test...